bitbots_body_behavior/bitbots_body_behavior/body_behavior.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+"""
+BehaviorModule
+^^^^^^^^^^^^^^
+.. moduleauthor:: Martin Poppinga <1popping@informatik.uni-hamburg.de>
+
+Starts the body behavior
+"""
+
+import os
+
+import rclpy
+from rclpy.node import Node
+
+from actionlib_msgs.msg import GoalID
+from bitbots_msgs.action import Dynup 
+from tf2_geometry_msgs import PoseStamped
+from humanoid_league_msgs.msg import GameState, HeadMode, Strategy, TeamData,\
+    RobotControlState, PoseWithCertaintyArray
+#from nav2_msgs.action import NavigateToPose
+from std_msgs.msg import Bool
+from visualization_msgs.msg import Marker
+
+from bitbots_blackboard.blackboard import BodyBlackboard
+from dynamic_stack_decider import dsd
+from geometry_msgs.msg import PoseWithCovarianceStamped, TwistWithCovarianceStamped, Twist
+from sensor_msgs.msg import PointCloud2
+from std_msgs.msg import Float32
+
+def main(args=None):
+    rclpy.init(args=None)
+    node = Node("body_behavior", automatically_declare_parameters_from_overrides=True)
+
+    blackboard = BodyBlackboard(node)
+    D = dsd.DSD(blackboard, 'debug/dsd/body_behavior')
+
+    D.blackboard.team_data.strategy_sender = node.create_publisher(Strategy, "strategy", 2)
+    D.blackboard.team_data.time_to_ball_publisher = node.create_publisher(Float32, "time_to_ball", 2)
+    D.blackboard.blackboard.head_pub = node.create_publisher(HeadMode, "head_mode", 10)
+    D.blackboard.pathfinding.direct_cmd_vel_pub = node.create_publisher(Twist, 'cmd_vel', 1)
+    D.blackboard.pathfinding.pathfinding_pub = node.create_publisher(PoseStamped, 'move_base_simple/goal', 1)
+    D.blackboard.pathfinding.pathfinding_cancel_pub = node.create_publisher(GoalID, 'move_base/cancel', 1)
+    D.blackboard.pathfinding.ball_obstacle_active_pub = node.create_publisher(Bool, "ball_obstacle_active", 1)
+    D.blackboard.pathfinding.keep_out_area_pub = node.create_publisher(PointCloud2, "keep_out_area", 1)
+    D.blackboard.pathfinding.approach_marker_pub = node.create_publisher(Marker, "debug/approach_point", 10)
+    D.blackboard.dynup_cancel_pub = node.create_publisher(GoalID, 'dynup/cancel', 1)
+    D.blackboard.hcm_deactivate_pub = node.create_publisher(Bool, 'hcm_deactivate', 1)
+
+    dirname = os.path.dirname(os.path.realpath(__file__))
```

Comment:
actions and descisions should be copied to the share install dir and accessed there.